zerver/lib/upload.py

```diff
@@ -599,7 +600,8 @@ def upload_export_tarball(self, realm: Optional[Realm], tarball_path: str,
         session = botocore.session.get_session()
         config = Config(signature_version=botocore.UNSIGNED)
```

Comment:
Noticed this while reading this PR -- this `signature_version` line seems a bit weird -- could we just use the same settings we use for other Boto access?

Reply:
This is required to obtain a public url without the parameters.